insights_messaging/consumers/kafka.py

```diff
@@ -2,10 +2,18 @@
 import os
 
 from confluent_kafka import Consumer as ConfluentConsumer
-from insights_messaging.consumers import Consumer, Requeue
+from insights_messaging.consumers import Consumer, Requeue, archive_context_var
 
 log = logging.getLogger(__name__)
 
+def update_archive_context_ids(payload):
+    if payload and "platform_metadata" in payload and 'host' in payload:
+        payload_id_dict = {}
+        payload_id_dict['dummy_key'] = 'dummy_value'
+        payload_id_dict['request_id'] = payload["platform_metadata"].get("request_id")
+        payload_id_dict['account'] = payload["platform_metadata"].get("account")
```

Comment:
Is it okay to log the account? Do other applications do it?

Reply:
OK. We can remove it.